indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -40,13 +40,15 @@
         var categoryRowSelector = 'tr[data-category-id]';
         var checkboxSelector = 'input[name=category_id]';
 
+
         $('.js-sort-categories').on('click', function() {
+            var sort_order = $(this).data('sort-order');
```

Comment:
`sortOrder` for the var JS, `sort-order` in the function argument - I'm also using the dash notation since quite some time there :)